tests/appsec/iast/test_iast_sql_injection.py

```diff
@@ -0,0 +1,46 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released
+
+# Weblog are ok for nodejs/express4 and java/spring-boot
+@coverage.basic
+@released(
+    dotnet="?",
+    golang="?",
+    java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.108.0", "spring-boot-openliberty": "0.108.0", "*": "?"},
+    nodejs={"express4": "3.6.0", "*": "?"},
+    php_appsec="?",
+    python="?",
+    ruby="?",
+    cpp="?",
+)
+class TestIastSqlInjection(BaseTestCase):
+    """Verify IAST features"""
+
+    EXPECTATIONS = {
+        "java": {"LOCATION": {"SQL_INJECTION": "com.datadoghq.system_tests.springboot.iast.utils.SqlExamples"}}
```

Comment:
Having the vulnerability in the location is redundant, as this file only covers SQLi